3.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+"""
+    The program print the number of rows and cols in the matrix
+"""
+arr = np.arange(15, dtype=np.int64).reshape(3, 5)
+print(arr)
+
+print(arr.shape)
```

Comment:
Your program should have a function that receives a matrix as an argument and returns it's cols & rows for EVERY matrix given. Rewrite your code with a function providing the correct functionality.